src/ir_Zepeal.cpp

```diff
@@ -0,0 +1,87 @@
+// Copyright 2020 Christian Nilsson (nikize)
+
+/// @file
+/// @brief Support for Zepeal protocol.
+/// This protocol uses fixed length bit encoding.
+/// Company behinde this brand seems to be Denkyosha
```

Comment:
Typo: behinde -> behind

Reply:
Forgot to do the last save (again) before commit, changed this test. (please check the updated text in a few)
embarrassing amount of issues in this PR, thanks for the awesome review!
... especially spelling

Reply:
Haha .. my speellang is atroachis. If I cann sport it yas noes its baid!